rdb/src/main/java/org/openmbee/sdvc/rdb/repositories/TWCIntegrationDAOImpl.java

```diff
@@ -0,0 +1,60 @@
+package org.openmbee.sdvc.rdb.repositories;
+
+import org.openmbee.sdvc.core.dao.TWCIntegrationDAO;
+import org.openmbee.sdvc.data.domains.global.TWCIntegration;
+import org.openmbee.sdvc.rdb.config.DatabaseDefinitionService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public class TWCIntegrationDAOImpl implements TWCIntegrationDAO {
```

Comment:
this is just from a preliminary glance of the pr, but we're reluctant to add project specific tables, is the need just to store more metadata from the project json? we have an open jira for saving the entire project/ref jsons (https://openmbee.atlassian.net/browse/MMS-56) that should eliminate the need for this, (there's also an existing Metadata table connected to Project, but we haven't decided which route to take yet)

Reply:
We missed the metadata table somehow.  Will refactor to use the metadata instead.